### PR TITLE
Make tests platform independent

### PR DESCRIPTION
**Pull request by logineimer**
I managed to get the tests running with the default Maven structure (`src/test/resources`). Unfortunately, I had to copy `UrlUtil`class into each project because Maven doesn't provide test code in the `fluentlenium-core` dependency.

Could you please verify the changes?

### CODE REVIEW

File: fluentlenium-festassert/src/test/java/org/fluentlenium/integration/util/UrlUtil.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package org.fluentlenium.integration.util;
+
+import java.net.URL;
+
+public class UrlUtil {
+
+    private UrlUtil() {
+        throw new UnsupportedOperationException("No instances allowed");
```
Comment (Toilal) [inline fluentlenium-festassert/src/test/java/org/fluentlenium/integration/util/UrlUtil.java]: making this class final should be OK without this exception.

**Comment by Toilal** [comment]
Please indent with 4 spaces instead of tabs

**Comment by Toilal** [comment]
I think we should get rid of the constants defined in `LocalFluentCase.java`, and just call getAbsoluteUrlFromFile directly from concrete tests. This would make them more readable ...

**Comment by Toilal** [comment]
But tests are running fine on my computer (ubuntu)

**Comment by logineimer** [comment]
I use Eclipse with spaces only formatter. It's awkward why Git thinks, there's a difference between yours and mine. Is it some Linux vs. Windows thing?

**Comment by logineimer** [comment]
Don't know if getting rid of the constants defined in LocalFluentCase is useful. What happens if the HTML file name changes? You would have to adapt all tests using the file instead of changing it in one place. However, this should be done in another pull request if necessary.

**Comment by logineimer** [comment]
I think, Travis crashed. :sweat_smile:

**Comment by Toilal** [comment]
Thanks @logineimer. Is the work over with this branch ? I'll try to give a try tomorrow.

**Comment by Toilal** [comment]
I've rebased everything in a single commit because commit history was weird, with branching and merging ... But it's merged, thanks.

**Pull request closed without merging**